packages/mui-material/src/styles/CssVarsProvider.tsx

```diff
@@ -54,4 +51,10 @@ function Experimental_CssVarsProvider(props: any) {
   return <CssVarsProvider {...props} />;
 }
 
-export { useColorScheme, getInitColorSchemeScript, CssVarsProvider, Experimental_CssVarsProvider };
+export {
+  useColorScheme,
+  getInitColorSchemeScript,
```

Comment:
Should we deprecate `getInitColorSchemeScript`? Or do we expect users to still use it?

Reply:
Good point. I think it can be removed but I'd do it in a separate PR and add note to the migration doc in the same PR.